sourced/ml/transformers/basic.py

```diff
@@ -8,6 +8,30 @@
 from sourced.ml.utils import EngineConstants, assemble_spark_config, create_spark
 
 
+class CsvSaver(Transformer):
+    def __init__(self, output, **kwargs):
+        super().__init__(**kwargs)
+        self.output = output
+
+    def __call__(self, head: RDD):
+        self._log.info("Writing %s", self.output)
+        return head.toDF() \
+            .repartition(1) \
+            .write \
+            .option("header", "true") \
+            .mode("overwrite") \
+            .csv(self.output)
+
+
+class Mapper(Transformer):
```

Comment:
This is pretty generic, I don't like it because it perverts the idea behind transformers. You should write your custom classes in case you want to apply a specific function.